cdm/core/src/main/java/ucar/nc2/dataset/NetcdfDataset.java

```diff
@@ -130,11 +130,17 @@ public enum Enhance {
      * x<0 --> 0 x>0 --> 1
      */
     ApplyClassifier,
```

Comment:
should we remove `ApplyClassifier`, `ApplyNormalizer`, `ApplyStandardizer` and instead use `ApplyRuntimeLoadedEnhancements` for those classes since those are being runtime loaded?

Reply:
Yeah I kinda think so too, it seems like if anyone has those attributes in their catalogs they specifically would want them to be on, can't see a reason to leave that fine-level control in.